src/main/java/de/teamlapen/vampirism/items/ItemBloodPotion.java

```diff
@@ -76,7 +76,7 @@ public ItemStack onItemUseFinish(ItemStack stack, World worldIn, EntityLivingBas
             BloodPotions.applyEffects(stack, entityLiving);
         }
 
-        if (entityplayer == null || !entityplayer.isCreative()) {
+        if (entityplayer == null || !entityplayer.abilities.isCreativeMode) {
```

Comment:
Oh, I didn't know that one still existed, but what is the advantage of that one vs isCreative()